content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -50,12 +50,12 @@ Only Kubernetes v1.25 through to v1.27 recognise `UserNamespacesStatelessPodsSup
 {{</ note >}}
 
 The cluster that you're using **must** include at least one node that meets the
-[requirements](/docs/concepts/workloads/pods/user-namespaces/#before-you-begin)
+[requirements]( https://kubernetes.io/docs/concepts/workloads/pods/user-namespaces/)
 for using user namespaces with Pods.
 
 If you have a mixture of nodes and only some of the nodes provide user namespace support for
 Pods, you also need to ensure that the user namespace Pods are
-[scheduled](/docs/concepts/scheduling-eviction/assign-pod-node/) to suitable nodes.
+[scheduled](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/) to suitable nodes.
```

Comment:
Why this change?

Reply:
it mention in issue #44573  that they are outdated need to fix

Reply:
can you please explain me what exactly to change there

Reply:
> Why this change?
it was mention in issue #44573 that they are outdated and need to fix can you please me what exactly to change